server/migrations/migrations.go

```diff
@@ -13,14 +13,14 @@ import (
 //go:embed *.sql
 var migrations embed.FS
 
-func Run(pgConfig config.PostgresConfig) error {
+func Run(cfg config.PostgresConfig) error {
+	fmt.Printf("connecting to %s (%+v)\n", cfg.Url(), cfg)
```

Comment:
🥜I would prefer to log everything consistently and avoid `Printf`?
```go
func Run(cfg config.PostgresConfig, logger zerolog.Logger) error {
	logger.Info().Str("url", cfg.Url()).Msg("connecting to")
```

Moreover, I don't like the idea of logging/printing the plain url and the `cfg` containing sensitive information like the password.